timeless/restaurants/table_shapes/views.py

```diff
@@ -78,4 +64,5 @@ class Delete(views.DeleteView):
 
 
```

Comment:
@vryazanov I suggest turning `bp` into `BP` into the file, since we are dealing with a static variable.